src/lib/native.ts

```diff
@@ -343,8 +343,21 @@ export async function temp(content: string, prefix: string) {
     })
 }
 
-export async function move(from: string, to: string) {
-    return sendNativeMsg("move", { from, to }).catch(e => {
+export async function move(
+    from: string,
+    to: string,
+    overwrite: boolean,
+    cleanup: boolean,
+) {
+    const requireNativeMessengerVersion = "0.3.0"
```

Comment:
Note to self: don't merge this without setting this to the ultimately required version

Reply:
@bovine3dom `nativegate` for version `0.3.0` was added in https://github.com/gsbabil/tridactyl_native_messenger/commit/043488b3cff5943678ed62bbbddda7ca3bf90ad6. Thanks.

Reply:
@bovine3dom Better Python/Nim compatibility added in https://github.com/gsbabil/tridactyl/commit/c17ac418266995de2f2540ee1178ee79eb41074a